setup.cfg

```diff
@@ -37,10 +37,13 @@ install_requires =
 
     # default numpy requirements
     numpy==1.13.3; python_version=='3.5' and platform_machine!='aarch64' and platform_system!='AIX'
-    numpy==1.13.3; python_version=='3.6' and platform_machine!='aarch64' and platform_system!='AIX'
-    numpy==1.14.5; python_version=='3.7' and platform_machine!='aarch64' and platform_system!='AIX'
-    numpy==1.17.3; python_version=='3.8' and platform_machine!='aarch64'
-    numpy==1.19.3; python_version=='3.9'
+    numpy==1.13.3; python_version=='3.6' and platform_machine!='aarch64' and platform_system!='AIX' and platform_python_implementation != 'PyPy'
+    numpy==1.14.5; python_version=='3.7' and platform_machine!='aarch64' and platform_system!='AIX' and platform_python_implementation != 'PyPy'
+    numpy==1.17.3; python_version=='3.8' and platform_machine!='aarch64' and platform_python_implementation != 'PyPy'
+    numpy==1.19.3; python_version=='3.9' and platform_python_implementation != 'PyPy'
+
+    numpy==1.19.0; python_version=='3.6' and platform_python_implementation=='PyPy'
+    numpy==1.20.0; python_version=='3.7' and platform_python_implementation=='PyPy'
```

Comment:
Can you change this line to 1.19.0, and add a comment to update to 1.20.0 once released?

Reply:
done